antlr_ast.py

```diff
@@ -2,25 +2,27 @@
 
 from ast import AST
 from antlr4.Token import CommonToken
-from antlr4 import InputStream, CommonTokenStream
+from antlr4 import CommonTokenStream
+from inputstream import CaseTransformInputStream
```

Comment:
Using `from antlr_ast.inputstream` or `from .inputstream` will clarify that `inputstream` isn't an external package. (I can't remember whether there are disadvantages, so would be interested to hear if there are).

Reply:
This kind of import isn't possible in this situation I think, as the two files are not in a package.